src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2330,11 +2330,13 @@ export function isSpecialChart(chartMeta: ChartMeta) {
 export function getChartSettingsMap(
     visibleAttributes: ChartMeta[],
     columns: number,
-    chartDimensionSet: { [uniqueId: string]: ChartDimension },
-    chartTypeSet: { [uniqueId: string]: ChartType },
-    genomicChartSet: { [id: string]: GenomicChart },
-    genericAssayChartSet: { [id: string]: GenericAssayChart },
-    clinicalDataBinFilter: { [uniqueId: string]: ClinicalDataBinFilter },
+    chartDimensionMap: { [uniqueId: string]: ChartDimension },
```

Comment:
Sorry, I wasn't clear here. I don't want these renamed at all. It's not a significant change, and it messes up the git blame.